lib/kvdb/kvdb_rparams.c

```diff
@@ -1433,6 +1433,51 @@ static const struct param_spec pspecs[] = {
             }
         },
     },
+    {
+        .ps_name = "storage.capacity.dio_disable",
```

Comment:
Change to storage.capacity.direct_io.enabled.

More verbose and matches the previous "enabled" "disabled" terminology we have going on

Reply:
Yes, I like that! Will change it to storage.capacity.directio.enabled and default it to true.

Reply:
Unfortunate, because "enable/disable" are directives, while "enabled/disabled" are observations of current state...

Reply:
You mean enable/disable is better than enabled/disabled for these rparams, right?

Reply:
In the old days, we tried to have `0` be the default value for all params (all params were scalar types).  So some were named `enable`, others were named `disable` depending on the desired default behavior.